### DISCUSSION TITLE
Unable to extract common name as a named entity

### DISCUSSION THREAD

**Discussion by mgrishaber-merlin** [explosion/spaCy#9568] [Help: Model Advice] [answered]
I am trying to extract the named entities for the following string
'Pope Benedict in a red hat'
I am expecting to extract 'Pope Benedict' as a single entity.

If I run it in the spacy.io interactive web app as follows:

```
import spacy

# Load English tokenizer, tagger, parser and NER
nlp = spacy.load("en_core_web_sm")

# Process whole documents
text = ("Pope Benedict in a red hat")
doc = nlp(text)

# Analyze syntax
print("Noun phrases:", [chunk.text for chunk in doc.noun_chunks])
print("Verbs:", [token.lemma_ for token in doc if token.pos_ == "VERB"])

# Find named entities, phrases and concepts
for entity in doc.ents:
    print(entity.text, entity.label_)
```

The name 'Benedict' is identified as a person, but the title of 'Pope' is not identified.

Noun phrases: ['Pope Benedict', 'a red hat']
Verbs: []
Benedict PERSON

If I repeat the test using the "en_core_web_trf" model, neither 'Pope' nor 'Benedict' is identified, which is unexpected

I am wondering if I need to provide additional information or maybe these are not the appropriate models?
spaCy Info is as follows:

## Info about spaCy

- **spaCy version:** 3.1.4
- **Platform:** Linux-5.4.0-89-generic-x86_64-with-debian-bullseye-sid
- **Python version:** 3.7.6
- **Pipelines:** en_core_web_sm (3.1.0)

Thanks for your help,
Mike

**Comment by polm** [comment, marked as answer]
Two issues here:

1. "Pope" is not included.

Whether to include titles or not is a decision made when designing training data. For the data in the spaCy models, titles are not included. 

2. "Benedict" is not always recognized as a name.

This is an accuracy problem, please understand the models are not perfect. It's also generally not possible to analyze why any specific single sentence failed. See #3052.
